website/docs/components/rich-tooltip/partials/code/component-api.md

```diff
@@ -0,0 +1,125 @@
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="text" @type="string">
+    Plain text string that will appear as `<span>` HTML element inside the toggle button. It has a dotted underline styling applied to it.
+  </C.Property>
+  <C.Property @name="icon" @type="string">
+    Use this parameter to show an icon. Any [icon](/icons/library) name is acceptable. The icon size is proportional to the font size (`1em`).
+    <br />
+    <br />
+    _If the toggle contains only the icon, an `aria-label` value needs to be assigned to the toggle to make it accessible._
+  </C.Property>
+  <C.Property @name="iconPosition" @type="enum" @values={{array "leading" "trailing"}} @default="trailing">
+    Positions the icon before or after the text.
+  </C.Property>
+  <C.Property @name="size" @type="string" @values={{array "undefined" "small" "medium" "large"}} @default="undefined">
+    If undefined (default), the `@text` inherits its typographic styles from the parent element.
+    <br />
+    If instead the `@size` is defined, a typographic style (and font size) is applied  to the `@text` according to the scale `body-100/200/300`.
+    <br />
+    <br />
+    _⚠️ Important: the size is applied only to the `@text`, not the `yield` content._
+  </C.Property>
+  <C.Property @name="isInline" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Sets the display for the `<button>` HTML element of the "toggle" content. If set to `true` it sets the element's display to `inline-flex` (a `<button>` can't have an `inline` layout).
+  </C.Property>
+  <C.Property @name="yield" @type="yield">
+    It's possible to yield generic content to the "toggle" element, instead of using the `@text` argument. This should be used only in special/edge cases where the toggle based on text/icon doesn't work for the specific context or design.
+    <br />
+    <br />
+    _⚠️ Important: in this case it's up to the consumers to make sure the implementation is compliant with the [accessibility requirements](/components/rich-tooltip?tab=accessibility)._
+  </C.Property>
+  <C.Property @name="...attributes">
+    This component supports use of [`...attributes`](https://guides.emberjs.com/release/in-depth-topics/patterns-for-components/#toc_attribute-ordering).
+  </C.Property>
+</Doc::ComponentApi>
+
+#### RichTooltip::Bubble
+
+Generic container that yields its content inside the "tooltip" popover element.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="yield">
+    Elements passed as children are yielded as inner content of the tooltip's bubble.
+    <br />
+    <br />
+    _⚠️ Important: we apply a CSS reset (`all: initial`) to the container, to avoid that styles applied to the parent elements leak into the tooltip content. If you find any issue with this reset please [speak with the Design System team](/about/support)._
```

Comment:
```suggestion
    _⚠️ Important: We apply a CSS reset (`all: initial`) to the container, to avoid  having styles applied to the parent elements leak into the tooltip content. If you find any issue with this reset, please [speak with the Design System team](/about/support)._
```